### DISCUSSION TITLE
https://developer.ted.europa.eu/ HTTP 403

### DISCUSSION THREAD

**Discussion by JanEisen** [OP-TED/eForms-SDK#218] [Q&A] [answered]
Whenever I try to open the TED developer production environment https://developer.ted.europa.eu/ the only response is a HTTP 403
```
403 ERROR
The request could not be satisfied.
Request blocked. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
Generated by cloudfront (CloudFront)
Request ID: uBSSncwMSYoZyN1_hzBjw7hYPfffY-RFQb5OL0pLCpHMCYkbbjw20A==
```

Am I doing something wrong or is there a timeline when this is expected to work?

**Comment by bertrand-lorentz** [comment, marked as answer]
The website was up and running, but a technical issue prevented access from the Internet. This has now been resolved.

In the future, for any problem related to our applications and websites, please contact our HelpDesk at: ted@publications.europa
